serde-tests/test.rs

```diff
@@ -29,14 +29,6 @@ use bson::{
     DeserializerOptions,
     Document,
     JavaScriptCodeWithScope,
-    RawArray,
```

Comment:
these tests had to be removed because they're implemented in a separate crate and thus only have access to the public API of `bson`